docs/contributing.rst

```diff
@@ -59,3 +67,35 @@ This will install:
 	- All the **style** modules, such as ``flake8``, to help you linter the code!
 	- All the **documentation** modules, like ``sphinx``, so that you can build the docs locally before submitting them.
 	- All the **test** modules, like ``pytest``, in order for you to test your code locally before committing it!
+
+Check your installation!
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Type the commands::
+
+	cd phys2bids/tests
+    pytest
```

Comment:
These are not aligned.

Reply:
I still don't see them aligned, did you commit and push the changes?
```suggestion
	cd phys2bids/tests
	pytest
```